client/src/pages/home/Home.jsx

```diff
@@ -47,9 +47,9 @@ function Home() {
 
     //upload files to server
     Axios.post(`${process.env.REACT_APP_SERVER_URL}/api/upload`, formData, {
+      withCredentials:true,
```

Comment:
Can you explain this?

Reply:
mentioning `withCredentials:true` will ensure that the cookies are sent to the server with every request.